### PR TITLE
Added swedish locale

### PR DESCRIPTION
**Pull request by emilkarl**

### CODE REVIEW

**Comment by greysteil** [comment]
Thanks Emil. @harrison can you review?

File: config/locales/sv.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      - Måndag
+      - Tisdag
+      - Onsdag
+      - Torsdag
+      - Fredag
+      - Lördag
+    times:
+      other: '%{count} gånger'
+      one: '%{count} gång'
+    until: 'till %{date}'
```
Comment (feelepxyz) [inline config/locales/sv.yml]: Should this be `tills %{date}`? `Tills 2015-04-4`

File: config/locales/sv.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    days_of_week: '%{segments} %{day}'
+    days_of_month:
+      other: '%{segments} dagar i månaden'
+      one: '%{segments} dagen i månaden'
```
Comment (feelepxyz) [inline config/locales/sv.yml]: `'%{segments} dag i månaden'`?
`dagen` would work if it's `Första dagen i månaden`, but weird with `En/1 dag i månaden`

File: config/locales/sv.yml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    days_of_year:
+      other: '%{segments} dagar på året'
```
Comment (feelepxyz) [inline config/locales/sv.yml]: `'%{segments} dagar om året'`?

File: config/locales/sv.yml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      one: '%{segments} dag på året'
```
Comment (feelepxyz) [inline config/locales/sv.yml]: `'%{segments} dag om året'`?

File: config/locales/sv.yml [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    each_month:
+      one: Varje månad
+      other: Var %{count} månad
+    each_year:
+      one: Varje år
+      other: Var %{count} år
+    'on': på den %{sentence}
+    in: 'i %{target}'
+    integer:
+      negative: '%{ordinal} till siste'
```
Comment (feelepxyz) [inline config/locales/sv.yml]: `sista`?

File: config/locales/sv.yml [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        -2: 2a från sista
+      ordinal: '%{number}%{ordinal}'
+      ordinals:
+        default: e
+        1: e
+        2: e
+        3: e
+        11: e
+        12: e
+        13: e
```
Comment (feelepxyz) [inline config/locales/sv.yml]: `1a` `2a`?
Would this be enough?

```
default: e
1: a
2: a
```

**Comment by feelepxyz** [comment]
@emilkarl sorry for such a slow review! Not entirely clued up on the context of a lot of those messages so some of my suggestions might be off. Let me know what you think.

File: config/locales/sv.yml (hunk 6 above)
Comment (emilkarl) [inline config/locales/sv.yml]: Yes

File: config/locales/sv.yml (hunk 5 above)
Comment (emilkarl) [inline config/locales/sv.yml]: Yes, sista är more correct I think.

File: config/locales/sv.yml (hunk 4 above)
Comment (emilkarl) [inline config/locales/sv.yml]: Yes, should be _om_ on both these.

**Comment by emilkarl** [comment]
Good suggestions, fixed them now.

**Comment by feelepxyz** [comment]
Nice work! :+1:

**Pull request merged**